server/src/main/java/org/apache/iotdb/db/sync/sender/transfer/SyncClient.java

```diff
@@ -171,7 +171,8 @@ public void verifySingleton() throws IOException {
    * @param lockFile lock file
    */
   private boolean lockInstance(File lockFile) {
-    try (final RandomAccessFile randomAccessFile = new RandomAccessFile(lockFile, "rw")) {
+    try {
+      final RandomAccessFile randomAccessFile = new RandomAccessFile(lockFile, "rw");
```

Comment:
randomAccessFile.close() will be not call?

Reply:
> randomAccessFile.close() will be not call?

Please review next 15 line of codes below this change.
randomAccessFile will be released when Process exit.

Reply:
![image](https://user-images.githubusercontent.com/29083697/134880873-4951282e-f7e4-4da5-add7-6a6ed4f9dfc3.png)

Reply:
@wangchao316   Has my reply answer your question?

Reply:
The first thread acquires the lock, addShutdownHook.
The second thread executes new RandomAccessFile, but tryLock() fails. Do you want to close RandomAccessFile?
We can't just rely on close in the hook. If there is no hook, there is no close file.
@jamber001

Reply:
> The first thread acquires the lock, addShutdownHook.
> The second thread executes new RandomAccessFile, but tryLock() fails. Do you want to close RandomAccessFile?
> We can't just rely on close in the hook. If there is no hook, there is no close file.
> @jamber001

I understand your thought, now.  
Though this will not affect sync-tool process, but from function view, it is better to close RandomAccessFile.
I will add codes for this.